From 320b8d5c0c858111e688abe400a899dcac315e43 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 23 Jun 2016 14:24:40 +0200
Subject: [PATCH] network/dhclient.conf: add missing commata

options need to be seperated by ,

(cherry picked from commit 7e51abc81f53c08e464decd4103e8c4ec25fef87)
---
 modules.d/40network/dhclient.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/40network/dhclient.conf b/modules.d/40network/dhclient.conf
index 7b067632..a1739ce3 100644
--- a/modules.d/40network/dhclient.conf
+++ b/modules.d/40network/dhclient.conf
@@ -3,4 +3,4 @@ option classless-routes code 121 = array of unsigned integer 8;
 
 request subnet-mask, broadcast-address, time-offset, routers,
         domain-name, domain-name-servers, domain-search, host-name,
-        root-path, interface-mtu classless-routes;
+        root-path, interface-mtu, classless-routes;
